test_metal_perf.py

```
@@ -0,0 +1,100 @@
+if __name__ == "__main__":
+    for test in TESTS:
+        m, n, k = test
+
+        for dtype in ("float32", "float16"):
+            torch_dtype = getattr(torch, dtype)
+
+            a = torch.randn(m, k, device="mps", dtype=torch_dtype)
+            b = torch.randn(k, n, device="mps", dtype=torch_dtype)
+            c = torch.zeros(m, n, device="mps", dtype=torch_dtype)
+
+            torch_add = lambda: torch.matmul(a, b, out=c)
```

Ruff (0.12.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `torch_add` as a `def`

(E731)